### PR TITLE
Rovil branch

### PR DESCRIPTION
**Pull request by kontchou-rovil**

### CODE REVIEW

File: conference.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# conference
```
Comment (eFunzo) [inline conference.py:1]: ce fichier n'est pas suffisamment espace

**Review by eFunzo — CHANGES_REQUESTED** [review]

File: devinette.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import random
+
+n = random.randint(0,100)
+
+appreciation = "?"
+
+while True:
+    var = input("Enter a number :")
+
+    var = int(var)
```
Comment (Yantiomene) [inline devinette.py:10]: this line can be merge with the previous one

**Review by Yantiomene — CHANGES_REQUESTED** [review]

File: devinette.py (hunk 2 above)
Comment (simogael) [reply devinette.py:10]: ofcourse even inside every condition's loop he can merge any two statements .

**Review by simogael — APPROVED** [review]
This code works great. It can still have some retouching.

**Pull request closed without merging**